client/my-sites/earn/memberships/index.jsx

```diff
@@ -206,27 +207,45 @@ class MembershipsSection extends Component {
 	}
 
 	renderSubscriberList() {
+		const isPayments = isEnabled( 'earn/rename-payment-blocks' );
 		return (
 			<div>
-				<SectionHeader label={ this.props.translate( 'Subscribers' ) } />
+				<SectionHeader label={ this.props.translate( 'Customers and Subscribers' ) } />
```

Comment:
Is this intentionally out of the feature flag check?

Reply:
Might have missed this one, I'll update

Reply:
Updated in https://github.com/Automattic/wp-calypso/pull/42848/commits/e398d7865bd36933fabb2c80424cbd6a959570d5